docs/development.md

```diff
@@ -22,7 +15,27 @@ for new developers picking up your work.
 
 ## Authentication
 
-Supply details here for information on how to authenticate with a provider so
-that developers have an idea of what's needed to hit APIs. It may be useful to
-provide explanations for each value specified in
-[../src/instanceConfigFields.json](../src/instanceConfigFields.json).
+The basic REST API uses just an ORG_URL and API_TOKEN. The org url will be in
+the format https://yoursubdomain.okta.com. The API token is generated from an
+Admin account on Okta.
+[Documentation](https://developer.okta.com/docs/api/getting_started/getting_a_token)
+
+Okta supports OAuth also, but the standard integration template is not setup for
+that.
```

Comment:
Yah, this will be something we need to work out later, among a few other integrations that could do OAuth.